cmd/run.go

```diff
@@ -711,6 +723,9 @@ func bindRunFlags(cmd *cobra.Command) {
 
 	defaultConfig := DefaultConfig()
 
+	cmd.Flags().StringSlice("enable-experimentals", defaultConfig.ExperimentalsEnabled, "a list of experimental features to enable")
```

Comment:
```suggestion
	cmd.Flags().StringSlice("enable-experimentals", defaultConfig.ExperimentalsEnabled, "a list of experimental features to enable")
```

`enable-experimentals` sounds like a boolean. `experimentals-enabled` ?

Reply:
Renamed to `--experimentals`